src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private static final String NAME = "trapezoid";
+    private final int height;
+    private final int bottom;
+    private final int top;
+
+    public IsoscelesTrapezoid(Colors color, int height, int bottom, int top) {
+        super(color);
+        this.height = height;
+        //if random bottom is smaller than top
+        if (bottom >= top) {
+            this.bottom = bottom;
+            this.top = top;
+        } else {
+            this.bottom = top;
+            this.top = bottom;
+        }

```

Comment:
It is not the best practice to implement any logic inside the constructor. 
The main purpose of a constructor is to create an instance of a class, other things violate the Single Responsibility Principle.